### PR TITLE
Setup GitHub actions

### PR DESCRIPTION
**Pull request by MaxSac**

### CODE REVIEW

**Review by Jean1995 — CHANGES_REQUESTED** [review]
I believe we do not test the python interface at the moment, we are not even building it, right? We should change that before merging.

File: .github/workflows/pypi_depoloy.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+on:
+  page_build:
+  release:
+    types:
+      - created
+
+jobs:
+
+    build-n-publish:
+      name: Build and publish Python 🐍 distributions 📦 to PyPI and TestPyPI
```
Comment (Jean1995) [inline .github/workflows/pypi_depoloy.yml:12]: we do not distribute to TestPyPI anymore

File: .github/workflows/pypi_depoloy.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      - uses: actions/checkout@master
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Install build
+        run: python -m pip install build
+      - name: Build source tarball
+        run: python -m build -s
+      - name: Publish distribution 📦 to Test PyPI
```
Comment (Jean1995) [inline .github/workflows/pypi_depoloy.yml:24]: see above

**Comment by MaxSac** [comment]
Closes #143

**Review by Jean1995 — APPROVED** [review]

**Pull request merged**